### PR TITLE
don't download here

### PR DESCRIPTION
**Pull request by esheldon**
modified the config also to have info needed for the downloader
(source_type), so we can use this same config for both

### CODE REVIEW

**Comment by esheldon** [comment]
I also plan to drop back to using desmeds classes for getting coadd information, because they are actively maintaned and have already diverged with new features

File: config_files/des-pizza-slices.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 campaign: Y3A1_COADD
+source_type: finalcut
```
Comment (beckermr) [inline config_files/des-pizza-slices.yaml]: Where is this used/set?

**Comment by beckermr** [comment]
I made an issue for this. Feel free to do it here if you want.

**Comment by esheldon** [comment]
its used by the downloader; we can use this config as-is to send to that code if this entry is put here.

File: config_files/des-pizza-slices.yaml (hunk 1 above)
Comment (beckermr) [reply config_files/des-pizza-slices.yaml]: Ahhhh. It is used for the downloaded. NM

**Comment by esheldon** [comment]
I'll merge this one and do a separate PR against that issue

**Pull request merged**